### PR TITLE
Pre commit hook and lint/test script separation

### PR DESCRIPTION
**Pull request by nickbalestra**
Addresses:

4 in #10

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,10 +21,15 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "lint": "eslint .",
```
Comment (mattiaerre) [inline package.json]: I think we need to add a .eslintignore w/ the following content:

```
coverage/*
dist/*
```

could you please add that?

Comment (nickbalestra) [reply package.json]: Good point, do you prefer to have an `.eslintignore` file or simplu use the `--ignore-pattern` flag inside the `lint` script now that is decoupled from the rest?

Comment (mattiaerre) [reply package.json]: lets add the --ignore-pattern inside the npm script so we save us yet another config file 👍

Comment (nickbalestra) [reply package.json]: Coolio, done.

**Comment by mattiaerre** [comment]
LGTM I ❤️  it

**Pull request merged**